### PR TITLE
add wait argument to deploy()

### PR DESCRIPTION
**Pull request by dbasunag**

### CODE REVIEW

File: ocp_resources/resource.py [hunk 1]
```diff
@@ -463,8 +463,10 @@ def _sigint_handler(self, signal_received, frame):
         self.__exit__(exception_type=None, exception_value=None, traceback=None)
         sys.exit(signal_received)
 
-    def deploy(self):
+    def deploy(self, wait=False):
```
Comment (myakove) [inline ocp_resources/resource.py:466]: We have more resources that define wait, please add wait there as well.

**Review by myakove — CHANGES_REQUESTED** [review]

File: ocp_resources/resource.py (hunk 1 above)
Comment (dbasunag) [reply ocp_resources/resource.py:466]: I will update those resources.

File: ocp_resources/node_network_configuration_policy.py [hunk 2]
```diff
@@ -304,18 +304,21 @@ def apply(self, resource=None):
         for _ in samples:
             return
 
-    def deploy(self):
+    def deploy(self, wait=True):
```
Comment (myakove) [inline ocp_resources/node_network_configuration_policy.py:307]: wait should go to self.create() and not with if wait:

Comment (myakove) [inline ocp_resources/node_network_configuration_policy.py]: wait should go to self.create() and not with if wait:

**Review by myakove — CHANGES_REQUESTED** [review]

**Review by myakove — APPROVED** [review]

**Comment by dbasunag** [comment]
!verified

**Review by myakove — APPROVED** [review]

**Pull request merged**